diff --git a/node_modules/ink/build/ink.js b/node_modules/ink/build/ink.js
index 9a632f1..f429409 100644
--- a/node_modules/ink/build/ink.js
+++ b/node_modules/ink/build/ink.js
@@ -167,7 +167,7 @@ export default class Ink {
             this.fullStaticOutput += staticOutput;
         }
         if (this.lastOutputHeight >= this.options.stdout.rows) {
-            this.options.stdout.write(ansiEscapes.clearTerminal + this.fullStaticOutput + output);
+            this.options.stdout.write(ansiEscapes.cursorTo(0, 0) + this.fullStaticOutput + output);
             this.lastOutput = output;
             this.lastOutputHeight = outputHeight;
             this.log.sync(output);
diff --git a/node_modules/ink/build/output.js b/node_modules/ink/build/output.js
index 59d2c97..a6a4610 100644
--- a/node_modules/ink/build/output.js
+++ b/node_modules/ink/build/output.js
@@ -138,7 +138,7 @@ export default class Output {
             .map(line => {
             // See https://github.com/vadimdemedes/ink/pull/564#issuecomment-1637022742
             const lineWithoutEmptyItems = line.filter(item => item !== undefined);
-            return styledCharsToString(lineWithoutEmptyItems).trimEnd();
+            return styledCharsToString(lineWithoutEmptyItems)
         })
             .join('\n');
         return {
