diff --git a/node_modules/ink/build/ink.js b/node_modules/ink/build/ink.js
index a9c444f..4598e02 100644
--- a/node_modules/ink/build/ink.js
+++ b/node_modules/ink/build/ink.js
@@ -119,7 +119,7 @@ export default class Ink {
             this.fullStaticOutput += staticOutput;
         }
         if (outputHeight >= this.options.stdout.rows) {
-            this.options.stdout.write(ansiEscapes.clearTerminal + this.fullStaticOutput + output);
+            this.options.stdout.write(ansiEscapes.cursorTo(0, 0) + this.fullStaticOutput + output);
             this.lastOutput = output;
             return;
         }
diff --git a/node_modules/ink/build/output.js b/node_modules/ink/build/output.js
index 5f0dcf8..4bcf461 100644
--- a/node_modules/ink/build/output.js
+++ b/node_modules/ink/build/output.js
@@ -136,7 +136,7 @@ export default class Output {
             .map(line => {
             // See https://github.com/vadimdemedes/ink/pull/564#issuecomment-1637022742
             const lineWithoutEmptyItems = line.filter(item => item !== undefined);
-            return styledCharsToString(lineWithoutEmptyItems).trimEnd();
+            return styledCharsToString(lineWithoutEmptyItems)//.trimEnd();
         })
             .join('\n');
         return {
