diff --git a/node_modules/ink/build/ink.js b/node_modules/ink/build/ink.js
index ec11d33..fd0fb38 100644
--- a/node_modules/ink/build/ink.js
+++ b/node_modules/ink/build/ink.js
@@ -155,7 +155,7 @@ export default class Ink {
                     this.fullStaticOutput += staticOutput;
                 }
                 if (outputHeight >= this.options.stdout.rows) {
-                    this.options.stdout.write(ansiEscapes.clearTerminal + this.fullStaticOutput + output);
+                    this.options.stdout.write(ansiEscapes.cursorTo(0, 0) + this.fullStaticOutput + output);
                     this.lastOutput = output;
                     return;
                 }
